First task

moumouseque opened the pull request:
test

Commit:
generate new rails app

Commit:
finish base functionality

Commit:
add bootstrap

moumouseque committed:
add simple bootstrap disign

moumouseque committed:
add simple bootstrap disign v.2

moumouseque committed:
add simple bootstrap disign v.2

moumouseque committed:
add simple bootstrap disign v.3

davetoxa commented:
@770580 в vendor папке должно быть 2 файла, один css и один js файл, не надо всё всё пихать туда =)

moumouseque committed:
change bootstrap path

moumouseque committed:
kdkukukrkdkdkdkdchchankbkbkbkbkb:q


:wq



:q
:qikbuit


;;:q!

moumouseque committed:
change bootstrap path v2
# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
# On branch first-task
# Changes to be committed:
#   (use "git reset HEAD^1 <file>..." to unstage)
#
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap-theme.css
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap-theme.css.map
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap-theme.min.css
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap-theme.min.css.map
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap.css
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap.css.map
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap.min.css
#	deleted:    vendor/bootstrap-3.3.6-dist/css/bootstrap.min.css.map
#	deleted:    vendor/bootstrap-3.3.6-dist/fonts/glyphicons-halflings-regular.eot
#	deleted:    vendor/bootstrap-3.3.6-dist/fonts/glyphicons-halflings-regular.svg
#	deleted:    vendor/bootstrap-3.3.6-dist/fonts/glyphicons-halflings-regular.ttf
#	deleted:    vendor/bootstrap-3.3.6-dist/fonts/glyphicons-halflings-regular.woff
#	deleted:    vendor/bootstrap-3.3.6-dist/fonts/glyphicons-halflings-regular.woff2
#	deleted:    vendor/bootstrap-3.3.6-dist/js/bootstrap.js
#	deleted:    vendor/bootstrap-3.3.6-dist/js/bootstrap.min.js
#	deleted:    vendor/bootstrap-3.3.6-dist/js/npm.js
#

moumouseque committed:
Revert "change bootstrap path"

This reverts commit eaf6ee7ab36ce3ec94592c8a52be8821cd945735.

moumouseque committed:
Merge branch 'first-task' of github.com:770580/flashcards into first-task
Revers

moumouseque committed:
change bootstrap path v2
